README.md

```diff
@@ -49,12 +66,14 @@ Deploying this system can be done with:
 This two step deploy is a limitation of the current deployment system, as you can't use the CRDs in the same run you create them.
 
 ### Run a pipeline
-- `kubectl create -f example-user-manifests/pipelinerun.yaml`
+
```

Comment:
nit: We should probably change this to use the `toolforge build` cli soon enough